tests/neg/i6795-b.check

```diff
@@ -0,0 +1,6 @@
+-- [E083] Syntax Error: tests/neg/i6795-b.scala:1:11 -------------------------------------------------------------------
+1 |sealed def y: Int = 1 // error
+  |           ^
+  |           Modifier(s) sealed not allowed for method
```

Comment:
I think we should be consistent in how we quote these modifiers. If `erased` is quoted with backticks, this should be as well